pkg/accounting/db.go

```diff
@@ -50,22 +49,22 @@ type Rollup struct {
 type DB interface {
 	// LastTimestamp records the latest last tallied time.
 	LastTimestamp(ctx context.Context, timestampType string) (time.Time, error)
-	// SaveAtRestRaw records raw tallies of at-rest-data.
-	SaveAtRestRaw(ctx context.Context, latestTally time.Time, created time.Time, nodeData map[storj.NodeID]float64) error
-	// GetRaw retrieves all raw tallies
-	GetRaw(ctx context.Context) ([]*Raw, error)
-	// GetRawSince retrieves all raw tallies since latestRollup
-	GetRawSince(ctx context.Context, latestRollup time.Time) ([]*Raw, error)
-	// GetStoragenodeBandwidthSince retrieves all storagenode_bandwidth_rollup entires since latestRollup
-	GetStoragenodeBandwidthSince(ctx context.Context, latestRollup time.Time) ([]*StoragenodeBandwidthRollup, error)
-	// SaveRollup records raw tallies of at rest data to the database
+	// SaveSNStorageTallies records the storagenode at rest data and updates LastTimestamp
```

Comment:
Often shortening names shows that structure can show that program structure can be improved. For example if the database interfaces was one of:

```
type StoragenodeAccounting interface { }
type ProjectAccounting interface { }

or. 

type StorageDB interface { }
type BandwidthDB interface { }
```

Reply:
that is a good idea. I'll open another ticket for that.